extensions/terminal-suggest/src/completions/code.ts

```diff
@@ -355,6 +355,2817 @@ const codeCompletionSpec: Fig.Spec = {
 		...extensionManagementOptions('code'),
 		...troubleshootingOptions('code'),
 	],
+	subcommands: [
+		{
+			name: 'tunnel',
+			description: 'Create a tunnel that\'s accessible on vscode.dev from anywhere.Run`code tunnel --help` for more usage info',
+			subcommands: [
```

Comment:
This change is a little more involved that this. The generated one has _a lot_ of duplication. We want to remove the duplication and derive them from code.ts where possible. Also a lot of this is specific to the code-tunnel, for example `code status` opens a file in the regular CLI, but for `code-tunnel` (or the native `code` CLI) it prints info similar to `code --status` .